### PR TITLE
Fixes for Xamarin Build

### PR DESCRIPTION
**Pull request by Adron**
Added gitignore to remove userprefs and re-added the references with the fixed Xamarin linkages.

### CODE REVIEW

File: Source/ElasticLINQ/packages.config [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Newtonsoft.Json" version="5.0.6" targetFramework="net45" />
```
Comment (bradwilson) [inline Source/ElasticLINQ/packages.config:3]: Shouldn't this go away?

File: Source/ElasticLINQ.Test/packages.config [hunk 2]
```diff
@@ -1,7 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Newtonsoft.Json" version="5.0.6" targetFramework="net45" />
```
Comment (bradwilson) [inline Source/ElasticLINQ.Test/packages.config:3]: Shouldn't this go away?

File: Source/ElasticLINQ.Test/packages.config [hunk 3]
```diff
@@ -1,7 +1,9 @@
...
+  <package id="Newtonsoft.Json" version="6.0.1" targetFramework="net45" />
   <package id="NSubstitute" version="1.6.1.0" targetFramework="net45" />
```
Comment (bradwilson) [inline Source/ElasticLINQ.Test/packages.config:5]: Shouldn't this go away?

Comment (Adron) [inline Source/ElasticLINQ.Test/packages.config:5]: Actually yeah, not sure why it left both in. Mine just ended up with 6.0.1.

**Comment by Adron** [comment]
Any word on this. Is it ok to pull in?

**Comment by bradwilson** [comment]
I'd like a chance to look at what the breaking changes in 6.0 are first, but otherwise, yes. :)

**Comment by bradwilson** [comment]
Merged in at 5d49b74f810c17a3a95ac41132275bbc7f904377

**Comment by Adron** [comment]
Thanks @bradwilson

**Pull request closed without merging**